locked-nft/contracts/NFTLocker.cdc

```diff
@@ -83,6 +94,14 @@ pub contract NFTLocker {
         return false
     }
 
+    pub fun nftIsLocked(id: UInt64, nftType: Type): Bool {
+        if let lockedToken = (NFTLocker.lockedTokens[nftType]!)[id] {
+            return true
```

Comment:
Don't you need to check the `lockedUntil` here?

Reply:
Glad you asked, because my intention was to delete the token from NFTLocker.lockedTokens, so that the key does not exist & would not have a lockedUntil. But I found a bug that I fixed here.

nftIsLocked will return false when the token does not exist for the type. Added tests to support.

https://github.com/dapperlabs/studio-platform-smart-contracts/pull/38/files#diff-787282924cd2e33a1f8859a16e87a50889699486d711b3fae0720d52b147b7f7R143